tests/RuleSetTest.php

```diff
@@ -302,24 +302,31 @@ public function testRiskyRulesInSet(array $set, $safe)
 
     public function provideSafeSets()
     {
-        return array(
+        $sets = array(
             array(array('@PSR1' => true), true),
             array(array('@PSR2' => true), true),
             array(array('@Symfony' => true), true),
+            array(array('@Symfony:risky' => true), false),
+            array(array('@PHP56Migration:risky' => true), false),
             array(
                 array(
                     '@Symfony:risky' => true,
                     '@Symfony' => false,
                 ),
                 false,
             ),
-            array(
-                array(
-                    '@Symfony:risky' => true,
-                ),
-                false,
-            ),
         );
+
+        if (PHP_MAJOR_VERSION >= 7) {
```

Comment:
no, this will not mark tests as skipped on php5

Reply:
I know, but I can't skip only some tests in provider. Or can I?

Reply:
you need to make 2nd test method with 2nd provider, that will (only one of them) contain php version requirement

Reply:
I wanted to avoid copying test method, so extracted cases... well, simply look at the updated PR, made it few seconds before your comment.

Reply:
@kubawerlos 
I do know it would be nice to achieve that, yet, for us it's better to be aware that some tests were skipped as php is not new enough.